docs/one-click-integrations/github-integration.md

```diff
@@ -139,6 +139,58 @@ The table below lists the data that the GitHub integration collects from your Gi
 
 <sup><span id="deployment-teams">2</span></sup>: Adding a new team or changing the composition of a team on GitHub only affects new data from that moment onwards and doesn't have an immediate impact on the dashboards. Also, deleted teams on GitHub will continue to show in Pulse.
 
+## Which permissions does Pulse need from GitHub? {: id="gh-permissions"}
+
+Pulse requests only the necessary permissions from GitHub to collect changes and deployment data from the repositories in your organization and [keeps your information secure](https://security.codacy.com/). See below the detailed list of permissions.
+
+<table>
+  <colgroup>
+    <col width="20%"/>
+    <col width="20%"/>
+    <col width="60%"/>
+  </colgroup>
+  <thead>
+    <tr>
+      <th>Scope</th>
+      <th>Permissions</th>
+      <th>Description</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td colspan="3"><strong>Repository permissions:</strong></td>
+    </tr>
+    <tr>
+      <td>Pull requests</td>
+      <td>Read</td>
+      <td>Pulse retrieves pull request information to calculate several metrics presented on the dashboards. <a href="#collected-data">See the details here.</a></td>
+    </tr>
+    <tr>
+      <td>Contents</td>
+      <td>Read</td>
+      <td>Pulse retrieves tag information to support semantic versioning. For this, the Pulse GitHub App requires read permission on the contents of files and directories, as it's the same permission that applies to tags.</td>
+    </tr>
+    <tr>
+      <td>Issues</td>
+      <td>Read</td>
+      <td>Pulse retrieves issue information to get the top-level comments of pull requests.</td>
+    </tr>
+    <tr>
+      <td colspan="3"><strong>Organization permissions:</strong></td>
+    </tr>
+    <tr>
+      <td>Webhooks</td>
+      <td>Read & Write</td>
+      <td>Pulse creates webhooks for organization and repository events (<code>create</code>, <code>pull_request</code>, <code>pull_request_review</code>) to calculate metrics.</td>
```

Comment:
We can further unpack this paragraph for more visibility:

- organization webhooks are used to know when
    - repositories are added to or removed from our integration
    - the integration is suspended or uninstalled
- repository events
    - `create` - to know when tags are created (basically the same reason as 'Contents' above)
    - `pull_request` - to know when a pull request is created, edited or deleted (same reason as 'Pull Requests' above)
    - `pull_request_review` - to know when a review is created, edited or deleted (this is just for the 'Changes & Reviews' dashboard)
    
In general, the webhook permission and associated events are necessary to gather data in real time. Other permissions are for historical data (obtained only after the integration is created). Members permission is used for both historical and real-time.

You don't have to add all of this info, it's more of a FYI.

Reply:
Thank you so much for the detailed information! I updated this paragraph. See how it looks now.